packages/wc-shoelace/components/multiInstancesSelect.ts

```diff
@@ -1,6 +1,6 @@
 import type { GraphPointer } from 'clownface'
 import { SlSelect } from '@shoelace-style/shoelace'
-import { difference } from '@ngard/tiny-difference'
+import tiny from '@ngard/tiny-difference'
```

Comment:
This is a commonjs package and should be imported differently to work correctly in pure ESM environment. Webpack is too magical

Reply:
And... of course changed does not work in webpack anymore 🙄